### ISSUE TITLE
Job Queue creation failing with Value Conversion Error

### ISSUE THREAD

**Issue by VenelinMartinov** [pulumi/pulumi-aws#3075]
### What happened?

During an upgrade to v6, an error was encountered with `JobQueue`:

```
aws:batch:JobQueue (snowflake-admin-api):
    error: [AttributeName("arn")] Value Conversion Error: An unexpected error was encountered trying to build a value. This is always an error in the provider. Please report the following to the provider developer:

    Received unknown value, however the target type cannot handle unknown values. Use the corresponding `types` package type or a custom type that handles unknown values.

    Path: arn
    Target Type: string
    Suggested Type: basetypes.StringValue

  aws:cloudwatch:EventRule (snowflake-api-cron):
```

### Example

```
aws.batch.JobQueue(
            resource_name=id_,
            args=aws.batch.JobQueueArgs(
                name=args.name,  # same as the lambda function
                priority=1,
                compute_environments=[compute.arn],
                state='ENABLED',
            ),
            opts=child_opts,
        )
```

### Output of `pulumi about`

.

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by VenelinMartinov** [comment]
Working on reproducing the issue.

**Comment by t0yv0** [comment]
Should be fixed in v6.13.1.
